www/src/views/starter-library/index.js

```diff
@@ -27,15 +27,17 @@ class StarterLibraryPage extends Component {
             content={`Gatsby Starters: ${filtersApplied}`}
           />
           <meta
-            name="og:description"
+            property="og:description"
             content={`Gatsby Starters: ${filtersApplied}`}
           />
           <meta
             name="twitter:description"
             content={`Gatsby Starters: ${filtersApplied}`}
           />
-          <meta name="og:title" content={filtersApplied} />
-          <meta name="og:type" content="article" />
+          <meta property="og:title" content={filtersApplied} />
+          <meta property="og:image" content={`../../logo.svg`} />
```

Comment:
Yeah, so I'm pretty sure this won't work. This isn't _require_'d so this is going to remain a relative path. This would presume that two levels above `https://www.gatsbyjs.org/starters` would contain a logo.svg, which isn't the case, correct?

Reply:
Not sure which one we need to use here..